src/app/containers/Header/index.test.jsx

```diff
@@ -104,4 +108,33 @@ describe(`Header`, () => {
     const skipLink = document.querySelector("a[href$='#content']");
     expect(skipLink).toBeVisible();
   });
+
+  const scriptLinkSelector = 'a[data-variant]';
+
+  it('should not render script link for a service without variants', () => {
```

Comment:
Same with the tests here. We should be fine with removing the Cypress test above.